src/gui/src/settings/options-window.cpp

```diff
@@ -33,7 +33,8 @@
 #include "settings/token-settings-widget.h"
 #include "settings/web-service-window.h"
 #include "theme-loader.h"
-
+#include "viewer/zoom-window.h"	// Still needed?
```

Comment:
There are still references to ZoomWindow's enum, but those could be changed to a different namespace as I noted elsewhere.

Reply:
Yes, I strongly believe it would make sense to extract them to another file.
It would perfectly fit in "custom-buttons".

Reply:
The reason I didn't do that is because I wanted all the settings related to ZoomWindow to be accessible in the same file (zoom-window.h). It feels a little more sensible and easier to maintain when buttons need to be changed. I was also hoping to keep the constants in ZoomWindow's namespace but I didn't want to instantiate them redundantly. I think I commented somewhere about that, asking whether it would be appropriate to include them as static data members. Of course, we can put all that in custom-buttons.h if you prefer.